R/theme.r

```diff
@@ -380,7 +380,13 @@ theme <- function(line,
 
 # Combine plot defaults with current theme to get complete theme for a plot
 plot_theme <- function(x) {
-  defaults(x$theme, theme_get())
+  if (is.null(attr(x$theme, "complete"))) {
```

Comment:
When using the default theme. The cases for the three branches respectively are.
```R
ggplot(...) + geom_point()

ggplot(...) + geom_point() + theme_gray()

ggplot(...) + geom_point() + theme(...)
```